# Copyright (C) 2000 Free Software Foundation, Inc.
# Cleber Rodrigues <cleberrrjr@bol.com.br>, 2000.
#
msgid ""
msgstr ""
"Project-Id-Version: gnome-db\n"
"POT-Creation-Date: 2000-11-30 01:23+0100\n"
"PO-Revision-Date: 2000-11-13 22:28-02:00\n"
"Last-Translator: Cleber Rodrigues <cleberrrjr@bol.com.br>\n"
"Language-Team: Brazilian Portuguese <???>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=koi8-r\n"
"Content-Transfer-Encoding: 8-bit\n"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: lib/gda-client/gda-connection-pool.c:267
#, c-format
msgid "could not open connection to %s"
msgstr "nao foi possível conectar a %s"

# fe/fe_main.c:205
#: lib/gda-client/gda-connection-pool.c:280
#, c-format
msgid "Data source %s not found"
msgstr "Fonte de dados %s não encontrada"

#: lib/gda-client/gda-connection.c:234 lib/gda-client/gda-error.c:179
#, c-format
msgid "%s: The server didn't respond."
msgstr "%s: O servidor não respondeu."

#: lib/gda-client/gda-connection.c:238 lib/gda-client/gda-error.c:183
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Um error ocorreu no sistema CORBA."

#: lib/gda-client/gda-connection.c:529
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "Banco de dados %s não encontrado nas configurações do sistema"

#: lib/gda-client/gda-connection.c:545
msgid "Could not open CORBA factory"
msgstr "Não foi possível abrir construtor CORBA"

# gda-clnt/gda-field.c:95
#: lib/gda-client/gda-field.c:245
msgid "<NULL>"
msgstr "<NULO>"

# gda-clnt/gda-field.c:101 gda-clnt/gda-field.c:103
#: lib/gda-client/gda-field.c:251 lib/gda-client/gda-field.c:253
msgid "<Unknown GDA Type(NULL)>"
msgstr "<Tipo de dados GDA Desconhecido (NULO)>"

# gda-clnt/gda-field.c:117
#: lib/gda-client/gda-field.c:266
msgid "TRUE"
msgstr "TRUE"

# gda-clnt/gda-field.c:119
#: lib/gda-client/gda-field.c:267
msgid "FALSE"
msgstr "FALSE"

#: lib/gda-common/gda-corba.c:77
#, c-format
msgid "CORBA System exception: %s"
msgstr "Erro no Sistema Corba (exception): %s"

#: lib/gda-common/gda-corba.c:110
msgid "True"
msgstr "Verdadeiro"

#: lib/gda-common/gda-corba.c:110
msgid "False"
msgstr "Falso"

#: lib/gda-common/gda-init.c:59
msgid "Attempt to initialize an already initialized client"
msgstr "Tentativa de inicializar um cliente já inicializado"

#: lib/gda-common/gda-log.c:48
msgid "Saving log...x"
msgstr "Salvando log...x"

#. compose message
#: lib/gda-common/gda-log.c:69
msgid "ERROR: "
msgstr "ERRO: "

#: lib/gda-common/gda-log.c:69
msgid "MESSAGE: "
msgstr "MENSAGEM: "

#: lib/gda-common/gda-thread.c:211
msgid "thread is already running"
msgstr "processo (thread) já está em execução"

#: lib/gda-common/gda-xml-database.c:282
#, c-format
msgid "table %s already exists"
msgstr "tabela %s já existe"

#: lib/gda-common/gda-xml-database.c:446
#, c-format
msgid "%p is not a valid table"
msgstr "%p não é uma tabela válida"

#: lib/gda-server/gda-server-error.c:185
msgid "Could not get pointer to server implementation"
msgstr "Não foi possível obter ponteiro para a implementação do servidor"

#: lib/gda-server/gda-server-impl-connection.c:123
#: lib/gda-server/gda-server-impl-connection.c:132
#: lib/gda-server/gda-server-impl-connection.c:139
#: lib/gda-server/gda-server-impl-connection.c:148
#: lib/gda-server/gda-server-impl-connection.c:155
#: lib/gda-server/gda-server-impl-connection.c:164
#: lib/gda-server/gda-server-impl-connection.c:171
#: lib/gda-server/gda-server-impl-connection.c:180
#: lib/gda-server/gda-server-impl-recordset.c:126
#: lib/gda-server/gda-server-impl-recordset.c:135
#: lib/gda-server/gda-server-impl-recordset.c:142
#: lib/gda-server/gda-server-impl-recordset.c:157
#: lib/gda-server/gda-server-impl-recordset.c:166
#: lib/gda-server/gda-server-impl-recordset.c:173
#: lib/gda-server/gda-server-impl-recordset.c:182
#: lib/gda-server/gda-server-impl-recordset.c:189
#: lib/gda-server/gda-server-impl-recordset.c:198
#: lib/gda-server/gda-server-impl-recordset.c:205
#: lib/gda-server/gda-server-impl-recordset.c:213
#: lib/gda-server/gda-server-impl-recordset.c:222
#: lib/gda-server/gda-server-impl-recordset.c:229
#: lib/gda-server/gda-server-impl-recordset.c:237
#: lib/gda-server/gda-server-impl-recordset.c:245
#: lib/gda-server/gda-server-impl-recordset.c:273
#: lib/gda-server/gda-server-impl-recordset.c:281
#: lib/gda-server/gda-server-impl-recordset.c:289
#: lib/gda-server/gda-server-impl-recordset.c:297
#: lib/gda-server/gda-server-impl-recordset.c:305
#: lib/gda-server/gda-server-impl-recordset.c:314
#, c-format
msgid "%s not implemented"
msgstr "%s não implementada"

#: lib/gda-server/gda-server-impl-connection.c:272
#, c-format
msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
msgstr "impl_GDA_Connection_Open: estabelecendo conexão com DSN: %s"

#: lib/gda-server/gda-server-impl-recordset.c:407
#, c-format
msgid "%s: an error ocurred while fetching data"
msgstr "%s: um erro ocorreu ao obter dados"

#: lib/gda-server/gda-server-impl-recordset.c:446
#, c-format
msgid "%s: length = %d"
msgstr "%s: comprimento = %d"

#: lib/gda-server/gda-server-impl-recordset.c:459
#, c-format
msgid "%s: sending field description for '%s'"
msgstr "%s: enviando descrição do campo para '%s'"

#: lib/gda-server/gda-server-impl.c:195
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Iniciando provedor %s sem funções de implementação"

#: lib/gda-server/gda-server-impl.c:222
#: report/server/gda-report-server-main.c:120
msgid ""
"OAF doesn't know about our IID; indicates broken installation. Exiting..."
msgstr ""
"OAF não sabe sobre nosso IID; isso indica instalação problemática. Saindo..."

#: lib/gda-server/gda-server-impl.c:225
msgid ""
"Another instance of this provider is already registered with OAF. Exiting..."
msgstr "Outra instância desse provedor já está registrado com OAF. Saindo..."

#: lib/gda-server/gda-server-impl.c:228
msgid "Unknown error registering this provider with OAF. Exiting"
msgstr "Erro desconhecido registrando esse provedor com OAF. Saindo..."

#: lib/gda-server/gda-server-impl.c:232
#, c-format
msgid "Registered with ID = %s"
msgstr "Registrando com ID = %s"

#: lib/gda-server/gda-server-impl.c:314
#, c-format
msgid "CORBA system exception %s"
msgstr "Excepção tipo sistema CORBA %s"

#: lib/gda-server/gda-server-impl.c:317
#, c-format
msgid "CORBA user exception: %s"
msgstr "Excepção tipo usuário CORBA: %s"

#: lib/gda-server/gda-server-init.c:63 lib/gda-server/gda-server-init.c:69
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Sinal %d recebido, fechando programa..."

#: lib/gda-server/gda-server-init.c:73
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Sinal %d recebido. Ignorando"

#: lib/gda-server/gda-server-init.c:123
msgid "Attempt to initialize an already initialized provider"
msgstr "Tentativa de inicializar um provedor já inicializado"

#: providers/gda-interbase-server/gda-interbase-connection.c:261
#: providers/gda-ldap-server/gda-ldap-connection.c:272
#: providers/gda-mdb-server/gda-mdb-connection.c:360
#: providers/gda-mysql-server/gda-mysql-connection.c:387
#: providers/gda-odbc-server/gda-odbc-connection.c:609
#: providers/gda-odbc-server/gda-odbc-connection.c:642
#: providers/gda-oracle-server/gda-oracle-connection.c:426
#: providers/gda-postgres-server/gda-postgres-connection.c:779
#: providers/gda-sybase-server/gda-sybase-error.c:106
#, c-format
msgid "error '%s' at %s"
msgstr "erro '%s' em %s\""

#: providers/gda-interbase-server/gda-interbase-connection.c:266
#: providers/gda-interbase-server/gda-interbase-connection.c:267
#: providers/gda-ldap-server/gda-ldap-connection.c:277
#: providers/gda-ldap-server/gda-ldap-connection.c:278
#: providers/gda-mdb-server/gda-mdb-connection.c:364
#: providers/gda-mdb-server/gda-mdb-connection.c:365
#: providers/gda-mysql-server/gda-mysql-connection.c:391
#: providers/gda-mysql-server/gda-mysql-connection.c:392
#: providers/gda-odbc-server/gda-odbc-connection.c:611
#: providers/gda-odbc-server/gda-odbc-connection.c:612
#: providers/gda-odbc-server/gda-odbc-connection.c:644
#: providers/gda-odbc-server/gda-odbc-connection.c:645
#: providers/gda-oracle-server/gda-oracle-connection.c:431
#: providers/gda-oracle-server/gda-oracle-connection.c:432
#: providers/gda-postgres-server/gda-postgres-connection.c:786
#: providers/gda-postgres-server/gda-postgres-connection.c:787
#: providers/gda-sybase-server/gda-sybase-error.c:115
#: providers/gda-sybase-server/gda-sybase-error.c:116
msgid "Not available"
msgstr "Não disponível"

#: providers/gda-interbase-server/gda-interbase-connection.c:268
#: providers/gda-ldap-server/gda-ldap-connection.c:279
#: providers/gda-mdb-server/gda-mdb-connection.c:366
#: providers/gda-mysql-server/gda-mysql-connection.c:393
#: providers/gda-oracle-server/gda-oracle-connection.c:433
#: providers/gda-postgres-server/gda-postgres-connection.c:788
#: providers/gda-sybase-server/gda-sybase-error.c:117
msgid "error"
msgstr "erro"

#: providers/gda-ldap-server/main-ldap.c:69
msgid "Could not register LDAP provider implementation"
msgstr "Não foi possível registrar implementação de provedor LDAP"

#: providers/gda-interbase-server/gda-interbase-connection.c:260
#, c-format
msgid "Error code %ld"
msgstr "Erro código %ld"

#: providers/gda-interbase-server/main-interbase.c:70
msgid "Could not register INTERBASE provider implementation"
msgstr "Não foi possível registrar implementação do provedor INTERBASE"

#: providers/gda-mail-server/gda-mail-connection.c:108
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-mysql-server/gda-mysql-connection.c:251
#: providers/gda-oracle-server/gda-oracle-connection.c:295
#: providers/gda-postgres-server/gda-postgres-connection.c:511
#: providers/gda-sybase-server/gda-sybase-connection.c:269
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "SCHEMA_QTYPE %d não manuseado"

#: providers/gda-mail-server/main-mail.c:70
msgid "Could not register Mail provider implementation"
msgstr "Não foi possível registrar implementação do provedor de Mail"

# fe/fe_editor.c:185
#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:87
#: providers/gda-sybase-server/gda-sybase-command.c:120
#, c-format
msgid "executing command '%s'"
msgstr "executando comando '%s'"

# fe/fe_editor.c:185
#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:89
#, c-format
msgid "info '%s'"
msgstr "informação '%s'"

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#: providers/gda-mysql-server/gda-mysql-connection.c:148
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr ""
"%s:%d: Você não pode especificar um UNIX_SOCKET se você também especificar "
"um HOST e uma PORT(a). Por favor remove o UNIX_SOCKET, ou remova ambos as "
"opções para HOST e PORT(a)"

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr ""
"%s:%d: Você deve indicar o nome do arquivo contendo o banco de dados tipo MS "
"Access"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr "Abrindo conexão com nome de arquivo=%s, usuário=%s, senha=%s"

#: providers/gda-mdb-server/gda-mdb-connection.c:415
#: providers/gda-mysql-server/gda-mysql-connection.c:442
msgid "mysql_recset is NULL"
msgstr "mysql_recset é NULO (NULL)"

#: providers/gda-mdb-server/gda-mdb-connection.c:457
#: providers/gda-mysql-server/gda-mysql-connection.c:484
#: providers/gda-sybase-server/gda-sybase-connection.c:432
msgid "No table name given for SCHEMA_COLS"
msgstr "Não foi fornecido um nome de tabela para SCHEMA_COLS"

#: providers/gda-mdb-server/main-mdb.c:70
#: providers/gda-mysql-server/main-mysql.c:70
msgid "Could not register MySQL provider implementation"
msgstr "Não pude registrar implementação de provedor MYSQL"

#: providers/gda-mysql-server/gda-mysql-connection.c:162
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%s, port=%s, "
"unix_socket=%s"
msgstr ""
"Abrinco conexão com usuário=%s, senha=%s, host=%s, porta=%s, unix_socket=%s"

#: providers/gda-odbc-server/gda-odbc-connection.c:280
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: SCHEMA_QTYPE %d não manuseado \n"

#: providers/gda-odbc-server/gda-odbc-connection.c:626
#: providers/gda-odbc-server/gda-odbc-connection.c:627
#: providers/gda-odbc-server/gda-odbc-connection.c:628
#: providers/gda-odbc-server/gda-odbc-connection.c:659
#: providers/gda-odbc-server/gda-odbc-connection.c:660
#: providers/gda-odbc-server/gda-odbc-connection.c:661
msgid "no text"
msgstr "sem texto"

#: providers/gda-odbc-server/main-odbc.c:69
msgid "Could not register ODBC provider implementation"
msgstr "Não foi possível registrar implementação de provedor ODBC"

# fe/fe_log.c:92
#: providers/gda-oracle-server/gda-oracle-command.c:195
msgid "Could not allocate statement handle"
msgstr "Não foi possível alocar handle para esta declaração "

#: providers/gda-oracle-server/gda-oracle-command.c:199
msgid "command object being created without associated connection"
msgstr "objeto de comando sendo criado sem estar associado a uma conexão"

# fe/fe_editor.c:185
#: providers/gda-oracle-server/gda-oracle-command.c:226
#, c-format
msgid "Executing command '%s'"
msgstr "Executando comando '%s'"

#: providers/gda-oracle-server/gda-oracle-connection.c:113
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "erro em OCIHandleAlloc() HTYPE_SVCCTX"

#: providers/gda-oracle-server/gda-oracle-connection.c:116
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "erro em OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:119
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "erro em OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:379
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "Tipo de dado ORACLE desconhecido %ld"

#: providers/gda-oracle-server/gda-oracle-recordset.c:115
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "Tipo de dado ORACLE desconhecido em fill_field_values"

#: providers/gda-oracle-server/gda-oracle-recordset.c:150
msgid "recordset object being created without associated connection"
msgstr ""
"objeto de conjunto de registros (recordset) sendo criado sem estar associado "
"a uma conexão"

#: providers/gda-oracle-server/main-oracle.c:69
msgid "Could not register ORACLE provider implementation"
msgstr "Não foi possível registrar implementação de provedor ORACLE"

#: providers/gda-postgres-server/gda-postgres-connection.c:888
#: providers/gda-postgres-server/gda-postgres-connection.c:1000
msgid "postgres_recset is NULL"
msgstr "postgres_recset e NULO (NULL)"

#: providers/gda-postgres-server/main-postgres.c:69
msgid "Could not register PostgreSQL provider implementation"
msgstr "Não foi possível registrar implementação do provedor PostgreSQL"

#: providers/gda-sybase-server/gda-sybase-command.c:115
msgid "Connection seems to be dead"
msgstr "Conexão parece estar morta"

#: providers/gda-sybase-server/gda-sybase-command.c:135
msgid "Filling command structure failed\n"
msgstr "Preenchimento de estrtura de comando falhou\n"

#: providers/gda-sybase-server/gda-sybase-command.c:147
msgid "sending cmd failed"
msgstr "envio de cmd falhou"

#: providers/gda-sybase-server/gda-sybase-command.c:154
msgid "Allocating recordset failed"
msgstr "Alocação de conjunto de registros (recordset) falhou"

#: providers/gda-sybase-server/gda-sybase-command.c:164
msgid "Getting recordset userdata failed"
msgstr ""
"Obtenção de conjunto de registros (recordset) de dados de usuários falhou"

#: providers/gda-sybase-server/gda-sybase-command.c:222
msgid "Command execution finished, all results processed"
msgstr "Execução de comando concluída, todos resultados processados"

# fe/fe_editor.c:138
#: providers/gda-sybase-server/gda-sybase-command.c:226
msgid "Command failed"
msgstr "Comando falhou"

#: providers/gda-sybase-server/gda-sybase-command.c:230
msgid "Execution of resultless command succeeded"
msgstr "Execução de comando sem resultado foi bem sucedida"

#: providers/gda-sybase-server/gda-sybase-command.c:246
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_result() retornaram %x"

#: providers/gda-sybase-server/gda-sybase-command.c:256
msgid "No further results, command completly succeeded."
msgstr "Sem mais resultados, comando foi completamente bem sucedido"

#: providers/gda-sybase-server/gda-sybase-command.c:260
msgid "An error occured during command execution."
msgstr "Um erro ocorreu durante a execução do comando"

#: providers/gda-sybase-server/gda-sybase-command.c:267
msgid "Cmd execution returned unknown result, may be an error"
msgstr ""
"Execução de comando retornou resultado desconhecido, provavelmente é um erro"

#: providers/gda-sybase-server/gda-sybase-command.c:280
msgid "No data returned, freeing recordset structure"
msgstr ""
"Nada foi retornado, liberando estrutura de conjunto de registros (recordset) "

#: providers/gda-sybase-server/gda-sybase-command.c:311
#: providers/gda-sybase-server/gda-sybase-command.c:323
msgid "Freeing command structure failed"
msgstr "Liberação de estrutra do comando falhou"

#: providers/gda-sybase-server/gda-sybase-connection.c:149
msgid "Error allocating error structure"
msgstr "Eroo alocando estrutura de erro"

#: providers/gda-sybase-server/gda-sybase-connection.c:157
msgid "Not enough memory for sybase_Connection"
msgstr "Não há memória suficiente para sybase_Connection"

#: providers/gda-sybase-server/gda-sybase-connection.c:422
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: table_name '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:438
msgid "Could not allocate command structure"
msgstr "Não foi possível alocar estrutura do comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:460
msgid "Could not allocate enough memory for query"
msgstr "Não foi possível alocar memória suficiente para consulta"

#: providers/gda-sybase-server/gda-sybase-connection.c:474
msgid "Could not get recordset in schema_cols"
msgstr "Não foi possível obter conjunto de registros em schema_cols"

#: providers/gda-sybase-server/gda-sybase-connection.c:506
#: providers/gda-sybase-server/gda-sybase-connection.c:583
#, c-format
msgid "schema_tables: table name = '%s'"
msgstr "schema_tables: nome da table = '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:514
#: providers/gda-sybase-server/gda-sybase-connection.c:591
#, c-format
msgid "schema_tables: table_schema = '%s'"
msgstr "schema_tables: table_schema = '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:519
#, c-format
msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
msgstr "schema_procedures: proc_catalog = '%s' NÃO USADO!\n"

#: providers/gda-sybase-server/gda-sybase-connection.c:523
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr "schmea_procs: tipo inválido de limitação %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:533
msgid "Could not allocate cmd structure"
msgstr "Não foi possível alocar estrutura cmd"

#: providers/gda-sybase-server/gda-sybase-connection.c:596
#, c-format
msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
msgstr "schema_procedures: proc_catalog = '%s' NÃO USADO!"

#: providers/gda-sybase-server/gda-sybase-connection.c:600
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr "schema_tables: tipo inválido de limitação %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:661
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr "Tipo de limitação inválido ou não suportado: %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:702
msgid "Could not allocate Gda_ServerCommand"
msgstr "Não foi possível alocar Gda_ServerCommand"

#: providers/gda-sybase-server/gda-sybase-connection.c:732
#: providers/gda-sybase-server/gda-sybase-connection.c:737
#, c-format
msgid "connection status: %x"
msgstr "status da conexão: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:733
#, c-format
msgid "return code: %x"
msgstr "código de retorno: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:770
msgid "attempting reconnection"
msgstr "tentando reconectar"

#: providers/gda-sybase-server/gda-sybase-connection.c:909
#, c-format
msgid "Reconnected to '%s'"
msgstr "Reconectado a '%s'"

# fe/fe_connection.c:46
#: providers/gda-sybase-server/gda-sybase-connection.c:911
#, c-format
msgid "Connected to %s"
msgstr "Conectado a %s"

#: providers/gda-sybase-server/gda-sybase-connection.c:919
#, c-format
msgid "Selected database '%s'"
msgstr "Banco de dados selecionado '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:948
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr "%s: Não foi possível alocar estrutura cmd"

#: providers/gda-sybase-server/gda-sybase-connection.c:957
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: Não foi possível alocar memória para string cmd"

#: providers/gda-sybase-server/gda-sybase-connection.c:970
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: ct_command falhou"

#: providers/gda-sybase-server/gda-sybase-connection.c:979
#, c-format
msgid "%s: sending command failed"
msgstr "%s: envio de comando falhou"

#: providers/gda-sybase-server/gda-sybase-connection.c:990
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: seleção de banco de dados falhou"

#: providers/gda-sybase-server/gda-sybase-connection.c:1008
#, c-format
msgid "%s: use command failed"
msgstr "%s: use de comando falhou"

#: providers/gda-sybase-server/gda-sybase-connection.c:1035
msgid "Could not allocate locale structure"
msgstr "Não foi possível alocar estrutura locale"

#: providers/gda-sybase-server/gda-sybase-connection.c:1041
#, c-format
msgid "Could not set locale to '%s'"
msgstr "Não foi possível mudar valor do locale para '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1047
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr "Não foi possível configurar locale para ser '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1052
msgid "Could not drop locale structure"
msgstr "Não foi possível remover estrutura do locale"

#: providers/gda-sybase-server/gda-sybase-connection.c:1065
#, c-format
msgid "Fatal error: %s\n"
msgstr "Erro fatal: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:62
#, c-format
msgid "gda_sybase_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-sybase-server/gda-sybase-error.c:71
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr ""
"Nenhuma flag de erro está ativada, mesmo assim %s foi chamada. Isto é "
"provavelmente um bug."

#: providers/gda-sybase-server/gda-sybase-error.c:135
msgid "cs_diag() initialized"
msgstr "cs_diag() inicializado"

#: providers/gda-sybase-server/gda-sybase-error.c:138
msgid "Could not initialize cs_diag()"
msgstr "Não foi possível inicializar cs_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:145
msgid "ct_diag() initialized"
msgstr "ct_diag() inicializado"

#: providers/gda-sybase-server/gda-sybase-error.c:148
msgid "Could not initialize ct_diag()"
msgstr "Não foi possível inicializar ct_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:182
#, c-format
msgid "%s could not find server implementation"
msgstr "%s não conseguiu localizar implementação do servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:188
#, c-format
msgid "%s could not get cnc userdata"
msgstr "%s não conseguiu obter dados do usuário para a conexão (cnc)"

#: providers/gda-sybase-server/gda-sybase-error.c:200
msgid "Could not get # of messages"
msgstr "Não foi possível obter número de mensagens"

#: providers/gda-sybase-server/gda-sybase-error.c:205
#, c-format
msgid "%d ctlib messages to fetch"
msgstr "%d mensagens da ctlib para obter-se"

#: providers/gda-sybase-server/gda-sybase-error.c:207
#, c-format
msgid "%d ctlib message to fetch"
msgstr "%d mensagem da ctlib para obter-se"

#: providers/gda-sybase-server/gda-sybase-error.c:243
msgid "error is nullpointer in gda_sybase_messages_handle"
msgstr "erro é ponteiro nulo em gda_sybase_messages_handle"

#: providers/gda-sybase-server/gda-sybase-error.c:268
#: providers/gda-sybase-server/gda-sybase-error.c:335
#: providers/gda-sybase-server/gda-sybase-error.c:378
msgid "Could not get # of server messages"
msgstr "Não foi possível obter número de mensagens do servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:276
#, c-format
msgid "Fetching %d client messages"
msgstr "Obtendo %d mensagens do cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:278
#, c-format
msgid "Fetching %d client message"
msgstr "Obtendo %d mensagem do cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:287
#, c-format
msgid "Could not get client msg # %d"
msgstr "Não foi possível obter mensagem (do cliente) número %d"

#: providers/gda-sybase-server/gda-sybase-error.c:303
msgid "Could not get # of client messages"
msgstr "Não foi possível obter número de mensagens do cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:311
msgid "Could not clear client messages"
msgstr "Não foi possível limpar mensagens do cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:343
#, c-format
msgid "Fetching %d server messages"
msgstr "Obtendo %d mensagens do servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:345
#, c-format
msgid "Fetching %d server message"
msgstr "Obtendo %d mensagem do servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:354
#, c-format
msgid "Could not get server msg # %d"
msgstr "Não foi possível obger mensagem (do servidor) número %d"

#: providers/gda-sybase-server/gda-sybase-error.c:386
msgid "Could not clear server messages"
msgstr "Não foi possível limpar mensagens do servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:408
#: providers/gda-sybase-server/gda-sybase-error.c:444
msgid "Could not get # of cslib messages"
msgstr "Não foi possível obter número de mensagens da cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:416
#, c-format
msgid "Fetching %d cslib messages"
msgstr "Obtendo %d mensagens da cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:418
#, c-format
msgid "Fetchong %d cslib message"
msgstr "Obtendo %d mensagem da cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:428
#, c-format
msgid "Could not get cslib msg # %d"
msgstr "Não foi possível obter mensagem (da cslib) número %d"

#: providers/gda-sybase-server/gda-sybase-error.c:452
msgid "Could not clear cslib messages"
msgstr "Não foi possível limpar mensagens da cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:584
msgid "Server message:"
msgstr "Mensagem do servidor:"

#: providers/gda-sybase-server/gda-sybase-error.c:593
msgid "Client message:"
msgstr "Mensagem do cliente:"

#: providers/gda-sybase-server/gda-sybase-error.c:600
msgid "CS-Library error:"
msgstr "Erro da CS-Library:"

#: providers/gda-sybase-server/gda-sybase-recordset.c:152
#, c-format
msgid "fetch error at row %d"
msgstr "Erro ao obter dados na coluna %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:159
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA na coluna %d; cancelando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:168
#, c-format
msgid "%s: Error (%d)"
msgstr "%s: Erro (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:343
msgid "Failed fetching # of columns"
msgstr "Obtenção de número de colunas falhou"

#: providers/gda-sybase-server/gda-sybase-recordset.c:349
#, c-format
msgid "Counted %d columns"
msgstr "Foram contadas %d colunas"

#: providers/gda-sybase-server/gda-sybase-recordset.c:356
#, c-format
msgid "%s could not allocate datafmt info"
msgstr "%s não pode alocar informação sobre datafmt"

#: providers/gda-sybase-server/gda-sybase-recordset.c:366
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s não pode alocar campos de dados"

#: providers/gda-sybase-server/gda-sybase-recordset.c:384
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe na coluna número %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:391
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "ct_describe na coluna %d falhou"

#: providers/gda-sybase-server/gda-sybase-recordset.c:397
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "ct_describe(%d): '%s', %d (%s)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:406
msgid "could not allocate data holder"
msgstr "não foi possível alocar armazenador para dados"

#: providers/gda-sybase-server/gda-sybase-recordset.c:416
msgid "could not ct_bind data holder to column"
msgstr "não foi possível ct_bind armazenador de dados para coluna"

#: providers/gda-sybase-server/gda-sybase-recordset.c:423
msgid "illegal type detected, aborting"
msgstr "tipo ilegar detectado, abortando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:429
msgid "could not allocate field"
msgstr "não foi possível alocar campo"

#: providers/gda-sybase-server/main-sybase.c:74
msgid "Could not register gda-sybase provider implementation"
msgstr "não foi possível registrar implementação do provedor gda-sysbase"

#: report/gda-report-engine.c:87
#, c-format
msgid "CORBA exception unloading report engine: %s"
msgstr "exceção CORBA, descarregando sistema de relatórios: %s"

#: report/server/gda-report-server-main.c:43
#: report/server/gda-report-server-main.c:50
#, c-format
msgid "Received signal %d, shutting down."
msgstr "Sinal %d recebido, fechando programa."

#: report/server/gda-report-server-main.c:54
#, c-format
msgid "Received signal %d, shutting down cleanly"
msgstr "Sinal %d recebido, fechando programa limpamente"

#: report/server/gda-report-server-main.c:96
msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
msgstr ""
"Falha ao inicializar OAF: por favor mande um relatório de bug aos "
"desenvolvedores do OAF"

#: report/server/gda-report-server-main.c:106
msgid "Could not activate report engine"
msgstr "Não foi possível ativar sistema de relatórios"

#: report/server/gda-report-server-main.c:123
msgid ""
"Another instance of the report engine is already registered with OAF. "
"Exiting..."
msgstr ""
"Outra instância do sistema de relatórios já está registrada com o OAF. "
"Saindo..."

#: report/server/gda-report-server-main.c:126
msgid "Unknown error registering report engine with OAF. Exiting"
msgstr "Erro desconhecido registrando sistema de relatórios com OAF. Saindo"

#: report/server/gda-report-server-main.c:130
#, c-format
msgid "Registered report engine with ID = %s"
msgstr "Sistema de relatórios registrado com ID = %s"

# fe/fe_main.c:186
#: tools/gda-run.c:34
msgid "database name"
msgstr "nome da banco de dados"

#: tools/gda-run.c:34
msgid "DATABASE"
msgstr "BANDO DE DADOS"

# fe/fe_impexp.c:164 fe/fe_impexp.c:274 fe/fe_impexp.c:391
# gda-clnt-ui/gnome-db-browser.c:217
#: tools/gda-run.c:35
msgid "user name"
msgstr "nome do usuário"

#: tools/gda-run.c:35
msgid "USER-NAME"
msgstr "NOME DO USUÁRIO"

#: tools/gda-run.c:36
msgid "password"
msgstr "senha"

#: tools/gda-run.c:36
msgid "PASSWORD"
msgstr "SENHA"

# fe/fe_impexp.c:164 fe/fe_impexp.c:274 fe/fe_impexp.c:391
# gda-clnt-ui/gnome-db-browser.c:217
#: tools/gda-run.c:37
msgid "file name"
msgstr "nome do arquivo"

#: tools/gda-run.c:37
msgid "FILE-NAME"
msgstr "NOME DO ARQUIVO"

#: tools/gda-run.c:109
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: entrada DSN mal configurada"

#: tools/gda-run.c:124
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: fonte de dados '%s' não encontrada\n"

#~ msgid "%s: not implemented"
#~ msgstr "%s: não implementada"
